src/js/components/bulkDownload/awards/filters/dateRange/TimePeriodFilter.jsx

```diff
@@ -97,6 +97,11 @@ export default class TimePeriodFilter extends React.Component {
                 newState.endDateUI = null;
             }
         }
+
+        if ((nextProps.filterTimePeriodEnd === '' && nextProps.filterTimePeriodStart === '') && nextProps.valid) {
```

Comment:
Is this necessary? Doesn't the parent `AwardDataContent` already run `validateForm` on `componentWillReceiveProps` anyway?

Reply:
it will run `validateForm` but when you hit reset it doesnt actually set the `validDate` state to false so it will allow you to submit a request without selecting a date on the second attempt